django_migration_vis/management/commands/visualizemigrations.py

```diff
@@ -32,7 +39,26 @@ def _create_digraph(self):
             self._add_dependencies(node)
 
     @staticmethod
-    def _style_label(tupled_node):
+    def _censor(text):
+        res = []
```

Comment:
do you want to fix the random seed? Isn't it make sense that the censored version also shows always the same result?